plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -59,12 +59,15 @@ public abstract class BaseBigQueryConnectorTest
     protected BigQuerySqlExecutor bigQuerySqlExecutor;
     private String gcpStorageBucket;
 
+    private String bigQueryConnectionId;
+
     @BeforeAll
     public void initBigQueryExecutor()
     {
         this.bigQuerySqlExecutor = new BigQuerySqlExecutor();
         // Prerequisite: upload region.csv in resources directory to gs://{testing.gcp-storage-bucket}/tpch/tiny/region.csv
         this.gcpStorageBucket = System.getProperty("testing.gcp-storage-bucket");
+        this.bigQueryConnectionId = System.getProperty("testing.bigquery-connection-id");
```

Comment:
Could you add `requireNonNull` here or within `testBigLakeExternalTable`?

Reply:
Done